docs/contributing/authors.md

```diff
@@ -306,24 +306,11 @@ You can search the [Pygments issue tracker](https://github.com/pygments/pygments
 
 ## Synchronize the Browser While Editing
 
-Use gulp to view changes in the browser while editing documentation.
 
-Install the gulp command line utility.
+Use `sphinx-autobuild` view the Makefile to view changes in the browser while editing documentation.
```

Comment:
I do not understand this sentence. Do you mean the following suggestion?
```suggestion
Use `sphinx-autobuild` to view changes in the browser while editing documentation.
```

Reply:
Accepted and committed.